scripts/deploy/azure/azure.tf

```diff
@@ -0,0 +1,170 @@
+    name = "webhint-staging"
+    location = "eastus"
+}
+
+resource "azurerm_app_service_plan" "linuxConsumptionPlan" {
+    name = "LinuxConsumptionPlan"
+    location = "westus"
+    resource_group_name = "${azurerm_resource_group.rg.name}"
+    kind = "Linux"
+    reserved = true
+
+    sku {
+        tier = "Dynamic"
+        size = "Y1"
+    }
+}
+
+resource "azurerm_app_service_plan" "linuxAppServicePlan" {
+    name = "LinuxAppServicePlan"
+    location = "${azurerm_resource_group.rg.location}"
+    resource_group_name = "${azurerm_resource_group.rg.name}"
+    kind = "Linux"
+    reserved = true
+
+    sku {
+        tier = "PremiumV2"
+        size = "P2v2"
+        capacity = 3
+    }
+}
+
+resource "azurerm_container_registry" "acr" {
+    name = "webhintstagingregistry"
+    resource_group_name = "${azurerm_resource_group.rg.name}"
+    location = "${azurerm_resource_group.rg.location}"
+    sku = "Basic"
+    admin_enabled = true
+}
+
+resource "azurerm_servicebus_namespace" "servicebus" {
+    name = "webhint-servicebus"
+    location = "${azurerm_resource_group.rg.location}"
+    resource_group_name = "${azurerm_resource_group.rg.name}"
+    sku = "Standard"
+}
+
+resource "azurerm_servicebus_queue" "jobsqueue" {
+    name = "webhint-jobs"
+    resource_group_name = "${azurerm_resource_group.rg.name}"
+    namespace_name = "${azurerm_servicebus_namespace.servicebus.name}"
+
+    dead_lettering_on_message_expiration = true
+    default_message_ttl = "P14D"
+    duplicate_detection_history_time_window = "PT30S"
+    enable_partitioning = false
+    lock_duration = "PT5M"
+    max_delivery_count = 30
+    max_size_in_megabytes = 1024
+}
+
+resource "azurerm_servicebus_queue" "resultsqueue" {
+    name = "webhint-results"
+    resource_group_name = "${azurerm_resource_group.rg.name}"
+    namespace_name = "${azurerm_servicebus_namespace.servicebus.name}"
+
+    dead_lettering_on_message_expiration = true
+    default_message_ttl = "P14D"
+    duplicate_detection_history_time_window = "PT30S"
+    enable_partitioning = false
+    lock_duration = "PT30S"
+    max_delivery_count = 10
+    max_size_in_megabytes = 1024
+}
+
+resource "azurerm_cosmosdb_account" "cosmosdb" {
+    name = "webhint-staging-database"
+    resource_group_name = "${azurerm_resource_group.rg.name}"
+    location = "${azurerm_resource_group.rg.location}"
+    offer_type = "standard"
+    kind = "MongoDB"
+
+    consistency_policy {
+        consistency_level = "session"
+    }
+
+    geo_location {
+        location = "${azurerm_resource_group.rg.location}"
+        failover_priority = 0
+    }
+}
+
+resource "azurerm_cosmosdb_mongo_database" "mongodb" {
+    name = "webhint"
+    resource_group_name = "${azurerm_resource_group.rg.name}"
+    account_name = "${azurerm_cosmosdb_account.cosmosdb.name}"
+
+    # This will be unnecessary once https://github.com/terraform-providers/terraform-provider-azurerm/pull/4467 is merged and published
```

Comment:
We cannot. The PR is merged, but it doesn't look like it is published

![image](https://user-images.githubusercontent.com/1581288/67724561-a9c53980-f99c-11e9-8a52-2b67b9086cc9.png)

Reply:
Looks like it will be in 1.36 once they publish.

Can you please open an issue to update the template once dependabot updates it?

Reply:
[Issue created](https://github.com/webhintio/serverless-online-service/issues/18), but we need to check every sprint, because we don't have any npm package for dependabot to update :(